geonode/settings.py

```diff
@@ -1028,7 +1028,7 @@
 FREETEXT_KEYWORDS_READONLY = False
 
 # notification settings
-NOTIFICATION_ENABLED = False or TEST
+NOTIFICATION_ENABLED = True or TEST
```

Comment:
@afabiani we enable them permanently now?

Reply:
@simod this is my idea, they works without no further setup and this option allows setup to create migrations at the beginning. If it's ok with you also I would merge the PR.

Reply:
+1 to merge if notifications now work